### DISCUSSION TITLE
Receive midi note and send to another channel Output

### DISCUSSION THREAD

**Discussion by Eddieso3** [djipco/webmidi#329] [Q&A]
Hello,

Its possible to receive a midi messsage from a input on channel 1 and send the same note on to a output but in another channel?
Like this
Input Note on Channel 1 --> Output Note on Channel 15

WebMidi.inputs[1].addForwarder(WebMidi.outputs[0])

Input 1 is one midi controller channel 1
output 2 is a Roland Synth that i want to receive message on channel15

**Comment by djipco** [comment]
The [`Input.addForwarder()`](https://webmidijs.org/api/classes/Input#addForwarder) method is designed to forward a whole [Input](https://webmidijs.org/api/classes/Input) to an [Output](https://webmidijs.org/api/classes/Output) without altering it. However, you can do what you are asking for relatively simply: 

```js
const inputChannel = WebMidi.inputs[1].channels[1];
const outputChannel = WebMidi.outputs[2].channels[15];

inputChannel.addListener("noteon", e => outputChannel.sendNoteOn(e.note));
```
